src/main/java/com/github/caldav4j/methods/HttpPropFindMethod.java

```diff
@@ -0,0 +1,246 @@
+	 * @param depth        Depth of the Propfind Method.
+	 * @throws IOException on error
+	 */
+	public HttpPropFindMethod(String uri, int propfindType, DavPropertyNameSet propNameSet, int depth) throws IOException {
+		super(uri, propfindType, propNameSet, depth);
+	}
+
+	/**
+	 * @param uri          Path of the principal
+	 * @param propfindType Type of Propfind Call. Specified, in DavConstants or CalDavConstants
+	 * @param depth        Depth of the Propfind Method.
+	 * @throws IOException on error
+	 */
+	public HttpPropFindMethod(String uri, int propfindType, int depth) throws IOException {
+		super(uri, propfindType, depth);
+	}
+
+	/**
+	 * Return the AclProperty relative to a given url
+	 *
+	 * @param urlPath Location of the CalendarResource
+	 * @param httpResponse Response Object for the request.
+	 * @return AclProperty xml response or null if missing
+	 * @author rpolli, ankushm
+	 */
+	public AclProperty getAcl(HttpResponse httpResponse, String urlPath) {
+		DavProperty p = getDavProperty(httpResponse, urlPath, CalDAVConstants.DNAME_ACL);
+		if (p != null) {
+			try {
+				return AclProperty.createFromXml(p.toXml(DomUtil.createDocument()));
+			} catch (DavException | ParserConfigurationException e) {
+				log.warn("Unable to create AclProperty");
+			}
+		}
+
+		return null;
+	}
+
+	/**
+	 * Return the ACL Ace returned from the PROPFIND call.
+	 * @param httpResponse Response Object for the request.
+	 * @param urlPath URL of the ACL
+	 * @return List of {@link AclProperty.Ace}
+	 * @throws CalDAV4JException on error retrieving them.
+	 */
+	public List<AclProperty.Ace> getAces(HttpResponse httpResponse, String urlPath) throws CalDAV4JException {
+		if (succeeded(httpResponse)) {
+			AclProperty acls = getAcl(httpResponse, urlPath);
+			return acls.getValue();
+		}
+		throw new CalDAV4JException("Error getting ACLs. PROPFIND status is: " + httpResponse.getStatusLine().getStatusCode());
+	}
+
+	/**
+	 * Convenience method to return the Calendar Description from the
+	 * @param httpResponse Response Object for the request.
+	 * @param urlPath Location of the CalendarResource
+	 * @return Calendar Description as String
+	 */
+	public String getCalendarDescription(HttpResponse httpResponse, String urlPath) {
+		DavProperty p = getDavProperty(httpResponse, urlPath, CalDAVConstants.DNAME_CALENDAR_DESCRIPTION);
+		if (p != null && p.getValue() != null) {
+			return p.getValue().toString();
+		} else {
+			return "";
+		}
+	}
+
+	/**
+	 * Convenience method to return the Calendar Display Name.
+	 * @param urlPath Location of the CalendarResource
+	 * @param httpResponse Response Object for the request.
+	 * @return Display Name as string
+	 */
+	public String getDisplayName(HttpResponse httpResponse, String urlPath) {
+		DavProperty p = getDavProperty(httpResponse, urlPath, DavPropertyName.DISPLAYNAME);
+		if (p != null && p.getValue() != null) {
+			return p.getValue().toString();
+		} else {
```

Comment:
In this simple cases I'd use:

if () {
  return
}
return "";

Reply:
Sure, I'll handle it.